sleap/nn/tracking.py

```
@@ -511,10 +520,149 @@ def get_candidates(
+        n_recent = deque(maxlen=max_length)
+        elapsed_recent = deque(maxlen=max_length)
+        last_report = time()
+        t0_all = time()
+        t0_batch = time()
+
+        for lf in frames:
+            new_lf = self.run_step(lf)
+
+            # Track timing and progress
+            elapsed_all = time() - t0_all
+            n_processed += 1
```

Ruff

Use `enumerate()` for index variable `n_processed` in `for` loop

(SIM113)